recipes-graphics/imx-gpu-viv/imx-gpu-viv-6.inc

```diff
@@ -304,7 +304,7 @@ FILES_libegl-imx-dev = "${includedir}/EGL ${includedir}/KHR ${libdir}/pkgconfig/
 
 FILES_libgal-imx = "${libdir}/libGAL${SOLIBS} ${libdir}/libGAL_egl${SOLIBS}"
 FILES_libgal-imx-dev = "${libdir}/libGAL${SOLIBSDEV} ${includedir}/HAL"
-RDEPENDS_libgal-imx += "kernel-module-imx-gpu-viv"
+RDEPENDS_libgal-imx += "kernel-module-imx-gpu-viv valgrind"
```

Comment:
The way it is, valgrind will be pulled in for all installs. Is this intended?

I'd put it as `RSUGGESTS_libgal-imx`.